src/pytesseract.py

```diff
@@ -70,6 +71,9 @@ def run_tesseract(input_filename,
 
     command += (tesseract_cmd, input_filename, output_filename_base)
 
+    if verbose:
+        command.append('tsv')
```

Comment:
Just a note about the output configs:
The official documentation states that the output config option is appended at the end of the cli command.
I am not sure if this is really the case, but I guess that for the sake of consistency with the official docs, maybe we can append the output config options at the end of the list with arguments?
(Also, in this way we may avoid any argument parsing bugs during the tesseract invocation)